conda/environments/cuspatial_dev_cuda11.2.yml

```diff
@@ -17,3 +17,4 @@ dependencies:
   - gtest=1.10.0
```

Comment:
We really need to add yml for more recent CUDA...  If these 3 files are all the same, we should probably just have one...  Should look at what cuDF does here.

Reply:
cuDF keeps only one version for 11.5
https://github.com/rapidsai/cudf/blob/ec79ce1b16d03920cd5da9123163428fc03f0d75/conda/environments/cudf_dev_cuda11.5.yml

Reply:
We should do that in a separate PR I guess.